plugin/wp-media-center.php

```diff
@@ -7,4 +7,23 @@
  * License URI:       https://www.gnu.org/licenses/gpl-2.0.html
  */
 
-echo 'Hello World';
+
+$environment = isset($_ENV['WP_MEDIA_CENTER_ENVIRONMENT']) ? $_ENV['WP_MEDIA_CENTER_ENVIRONMENT'] : 'production';
+$isDevelopment = $environment === 'development' ? true : false;
+
+function render_manager_page() {
+  echo('<div id="wp-media-center__manager-app"></div>');
+}
+
+function add_manager_scripts() {
+  global $isDevelopment;
+  $url = $isDevelopment ? 'http://localhost:8081/manager.js' : plugin_dir_url(__FILE__).'static/manager.js';
```

Comment:
In the `development` environment we use `webpack-dev-server` for `live-reload`. In any other environment we use  assets outputted to the `static` folder.